<launch>

	<!-- Participant ID (integer) -->
	<arg name="ID" default="0"/>
    
    <!-- Task ('basic' - standard trajectory from shelf to table, 'tilted' -
    same as basic, but the cup in the end effector is tilted) -->
	<arg name="task" default="basic"/> 

	<!-- Feature learning method (DISCRETE, CONTINUOUS)-->
	<arg name="feat_method" default="DISCRETE"/>

	<!-- List of string features the robot contains 
    ('efficiency', 'table', 'coffee', 'human', 'laptop', 'origin') -->
	<arg name="feat_list" default="table"/>

	<!-- List of feature weights the robot is initialized with (float). -->
	<arg name="feat_weight" default="0"/>

	<!-- Record data mode (T/F) -->
	<arg name="record" default="F"/>

	<!-- Simulate data mode (string of features to simulate) -->
	<arg name="simulate" default="None"/>	

    <!-- List of feature weights the robot simulates a demonstration with. -->
	<arg name="simulate_weight" default="0"/>

	<!-- Trajopt PID planning with learning from human demonstrations -->
	<node name="demo_control" pkg="beta_adaptive_pHRI" type="demo_control.py"
    args="$(arg ID) $(arg task) $(arg feat_method) $(arg feat_list) 
    $(arg feat_weight) $(arg record) $(arg simulate) $(arg simulate_weight)" output="screen"/>

</launch>

